examples/Devices/Technomancy/Atreus2/Atreus2.ino

```diff
@@ -99,25 +172,14 @@ KALEIDOSCOPE_INIT_PLUGINS(
   Focus,
   FocusEEPROMCommand,
   FocusSettingsCommand,
+  EscapeOneShot,
   OneShot,
   SpaceCadet,
   MouseKeys,
   Macros,
   Qukeys
```

Comment:
Qukeys should get registered before EscapeOneShot, OneShot, SpaceCadet, MouseKeys, and Macros. Of those others, SpaceCadet should be second, though I'd recommend dropping it and just using Qukeys if it's in there anyway.

Reply:
@gedankenexperimenter thanks for the suggestion! I tried implementing it as you said, and it seems a little bit better but still not bulletproof. I still get accidental triggers of the modifiers as I type.  Here's a gist of my ino file.  Please let me know if there's anything I might have wrong - https://gist.github.com/joeynguyen/c09a5f7cfed683e5db9af6c4cc5c0167